specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -3743,6 +3793,15 @@
         }
       }
     },
+    "BillingAccountUpdateProperties": {
+      "description": "The properties of the billing account that can be updated.",
+      "properties": {
+        "address": {
```

Comment:
One final thing:
The body of patch should replicate the resource but only include the fields being patched. In this case you have:
{
  'address':...
}
but it should be:
{
  'properties':{
    'address': ...
  }
}